daos-stack/daos
# DAOS-10858 test: dfuse multi-user perms with caching

Test-tag: pr DfuseMUPerms dfuse test_container_query_attr test_list_attrs_long fio soak_smoke test_container_user_acl test_daos_dfuse_unit DaosBuild
Skip-unit-tests: true
Skip-fault-injection-test: true

Skip-func-hw-test: true

- Verify multi-user dfuse perms with cache expiration
- Cleanup dfuse with a regiser_cleanup
- Add TestContainer.set_attr
- Rename container_set_attrs to container_set_attr and remove old container_set_attr

Required-githooks: true

### Before requesting gatekeeper:


### Gatekeeper:
